tests/e2e/package.json

```diff
@@ -23,8 +23,10 @@
     "cli-argument-parser": "0.4.5"
   },
   "devDependencies": {
+    "@types/chance": "^1.1.3",
```

Comment:
Please pin the versions

Reply:
Fixed. Why we should use exact version and “Compatible with version” is not valid?

Reply:
@ofersteinberg Why do we need to pin the version?

Reply:
We want to control it. It avoids failures in the future if they will have breaking changes.